hash/thinkpad8xx.xml

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0"?>
+<!DOCTYPE softwarelist SYSTEM "softwarelist.dtd">
+<!--
+license:CC0-1.0
+-->
+<softwarelist name="thinkpad8xx" description="IBM Thinkpad 800 series software">
+
+	<!-- Operating systems -->
+
+	<software name="aix_4_1_5">
+		<description>AIX 4.1.5</description>
+		<year>1997</year>
+		<publisher>IBM</publisher>
+		<sharedfeat name="compatibility" value="800,820,850" />
+		<part name="cdrom1" interface="cdrom">
+			<diskarea name="cdrom">
+				<disk name="aix_4.1.5_disc_01" sha1="4c49c8190e33c2c7f13a74d7332d384edd6b3042" />
+			</diskarea>
+		</part>
+		<part name="cdrom2" interface="cdrom">
+			<diskarea name="cdrom">
+				<disk name="aix_4.1.5_disc_01" sha1="228afedd90619e89c6b466b4970bda0e5a3e9a13" />
+			</diskarea>
+		</part>
+	</software>
+
+	<!-- Only supported on Thinkpad Series 820 and 850 -->
+	<software name="os2w_ppc">
+		<description>OS/2 Warp PowerPC Edition</description>
+		<year>1995</year>
+		<publisher>IBM</publisher>
+		<sharedfeat name="compatibility" value="820,850" />
+		<part name="cdrom1" interface="cdrom">
+			<feature name="part_number" value="19H1133"/>
+			<diskarea name="cdrom">
+				<disk name="aix_4.1.5_disc_01" sha1="b646fdb3ab5f982304e7cbaf6bd735e20cc66198" />
+			</diskarea>
+		</part>
+		<part name="cdrom2" interface="cdrom">
+			<feature name="part_number" value="19H1134"/>
+			<diskarea name="cdrom">
+				<!-- Applications Sampler December, 1995-->
+				<disk name="aix_4.1.5_disc_01" sha1="e1fd3631c353b2903d35658a59e8a79b6306c1d2" />
+			</diskarea>
+		</part>
+	</software>
```

Comment:
Is this OS/2 Warp, or AIX as the disc labels suggest?  If it’s actually OS/2 Warp, where did it come from?  OS/2 Warp wasn’t in the initial kits sent out with these notebooks.  I thought it was never officially released.  Also, remember to use `part_id`.

Reply:
![os2-ppc-02-cd2-sampler](https://github.com/user-attachments/assets/a1e722b7-95a0-427e-ad4a-68e594c135ec)
![os2-ppc-01-cd1-os](https://github.com/user-attachments/assets/9d2c6c0b-af67-4634-a665-7600e6ed1049)

I think it was a limited release, but there is no "beta" or "preview" on the disc label.

Fixed the CHD names, the SHAs were right. Thanks a lot.